pkgs/applications/networking/mailreaders/thunderbird/default.nix

```
@@ -332,5 +332,6 @@ stdenv.mkDerivation rec {
     ];
     platforms = platforms.linux;
     license = licenses.mpl20;
+    timeout = 28800; # eight hours
```

```suggestion
    timeout = 8 * 3600;
```

Why though? This seems pointless when the default is 10 hours on hydra.
If the timeout is for non-hydra builds, it can be changed directly on the release branch, because it doesn't cause a rebuild.

---

I don't think thunderbird can cause a mass rebuild in any case.  I'll try asking on the source of all this: https://github.com/NixOS/nixpkgs/pull/96767#issuecomment-724104413